Website/README.md

```diff
@@ -0,0 +1,10 @@
+# Website
+**In simple words, we can describe a README file as a guide that gives users a detailed description of a project you have worked on.**
+
+**It can also be described as documentation with guidelines on how to use a project. 
+Usually it will have instructions on how to install and run the project but we're already worked on it which is the main README file our project.**

```

Comment:
```suggestion
Usually it will have instructions on how to download, install, run and contribute to the project but we've already worked on it which is the main [README](https://github.com/SaptarshiSarkar12/Drifty/blob/master/README.md) file of our project.**
```